interface/patient_file/summary/demographics_full.php

```diff
@@ -384,6 +385,19 @@ function policykeyup(e) {
 <script> var constraints = <?php echo $constraints;?>; </script>
 
 <body class="body_top">
+
+<?php //Check if new patient is added to hooks
+//this will hold the javascript submit me param new_validate;
+$new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```

Comment:
Above variable is not used anywhere so can be removed. This is better so then we don't confuse the client side validation stuff with the zend stuff.